data.json

```
@@ -0,0 +1,61 @@
+[
+    {
+        "base": "فعل",
+        "forms": [
```

It is difficult to keep in mind all the combinations and exceptions. I think it is better to use pronouns as links like so:
```suggestion
        "forms": [
            {
                "tense": "past",
                "pronoun": "huwa",
                "text": { ...
```
I did not like this approach a first time we discussed it, but now it seems more reasonable for me.